src/v0/destinations/drip/util.js

```diff
@@ -33,7 +35,7 @@ const userExists = async (Config, id) => {
     if (response && response.status) {
       return response.status === 200;
     }
-    throw new CustomError("Invalid response.");
+    throw new NetworkInstrumentationError("Invalid response.");
```

Comment:
Is this clearly an instrumentation error?

Reply:
Yeah, it should be as it is the case when the response status is not 200.

Reply:
That's not the criteria for the network instrumentation error.